services/web-app/components/mdx-notification/mdx-notification.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { ActionableNotification, Button, Column, Grid, Link } from '@carbon/react'
+import { ChevronDown, ChevronUp } from '@carbon/react/icons'
+import clsx from 'clsx'
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+
+import styles from './mdx-notification.module.scss'
+
+export const MdxNotification = ({
+  content,
+  title,
+  description,
+  href,
+  link,
+  id,
+  kind = 'error',
+  collapsible = false
+}) => {
+  const wrapperClassName = clsx({
+    'cds--inline-notification': true
+  })
+
+  const [collapsed, setIsCollapsed] = useState(false)
+
+  return (
+    <Grid narrow className={styles['error-container']}>
+      <Column sm={4} md={6} lg={16}>
+        <ActionableNotification
+          href={href}
+          className={clsx(wrapperClassName)}
+          id={id}
+          inline
+          lowContrast
+          hideCloseButton
+          kind={kind}
+        >
+          {collapsible && (
+            <Button
+              className={styles['collapse-icon']}
+              kind="ghost"
+              size="lg"
+              onClick={() => {
+                setIsCollapsed(!collapsed)
+              }}
+              hasIconOnly
+              renderIcon={collapsed ? ChevronUp : ChevronDown}
+              iconDescription={collapsed ? 'Expand' : 'Collapse'}
+              tooltipAlignment="center"
+              tooltipPosition="bottom"
+            />
+          )}
+          <Grid className={styles.grid}>
+            <Column sm={4} md={6} lg={12}>
+              <span className={styles.container}>
+                <strong className={styles.title}>{title}</strong>
+              </span>
+              <p className={styles.description}>{description}</p>
+              {!collapsed && <p className={styles.content}>{content}</p>}
+            </Column>
+            <Column sm={4} md={6} lg={4}>
```

Comment:
```suggestion
            <Column sm={4} md={6} lg={12}>
              <span className={styles.container}>
                <strong className={styles.title}>{title}</strong>
              </span>
              <p className={styles.description}>{description}</p>
              {!collapsed && <p className={styles.content}>{content}</p>}
            </Column>
            <Column sm={4} md={6} lg={4}>
```

Unclear from redlines what the grid should be doing here, or if this should even be on the grid? I'm leaning towards no subgrid here and just flexbox, so it just leaves space for the link if there is one. Should check with design. 

I think we need some logic here also if there is or isn't a link? No link and it shouldn't have the empty space to the right?


![Screen Shot 2022-10-07 at 7 21 10 AM](https://user-images.githubusercontent.com/2753488/194552054-511a0405-bc13-46fd-9665-4b339d4d858f.png)